src/tinyformat.h

```diff
@@ -179,13 +179,19 @@ namespace tfm = tinyformat;
 
 namespace tinyformat {
 
+// Similar to std::runtime_format from C++26.
+struct RuntimeFormat {
+    explicit RuntimeFormat(std::string str) : fmt{std::move(str)} {}
+    std::string fmt;
```

Comment:
Noticed that the `std`-version uses `string_view`. Maybe add a note about why you didn't go with that in the commit message, or consider switching.